src/cli.rs

```diff
@@ -68,6 +68,15 @@ fn build_command() -> Command<'static> {
                 .help("Perform exactly NUM runs for each command. If this option is not specified, \
                        hyperfine automatically determines the number of runs."),
         )
+        .arg(
+            Arg::new("reference")
+                .long("reference")
+                .short('R')
```

Comment:
Let's not introduce a short command line option for this right away. We can always add it later if this is really a commonly used option.